.github/workflows/infra.yml

```diff
@@ -23,3 +23,31 @@ jobs:
     - run: npm ci
     - run: npm run build
     - run: npm run lint
+
+  build-driver-all64:
+    name: "build-driver-all64"
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 10.15
```

Comment:
nit: Its not fully related to this PR, but is there a reason why we are sometimes publishing with 10 and sometimes with 10.15?

Seems like currently only for Docker and linting 10.15 is used. So I would maybe use 10 which will result in the latest minor version.

Reply:
Done.